### DISCUSSION TITLE
Angular Primitives

### DISCUSSION THREAD

**Discussion by geromegrignon** [realworld-angular/realworld-angular#3] [Example apps candidates]
> Angular Primitives is a low-level headless UI component library with a focus on accessibility, customization, and developer experience. Whether you're building a robust design system from scratch or looking to enhance your existing one, our primitives are here to support you every step of the way.

GitHub repository: https://github.com/ng-primitives/ng-primitives/
Documentation: https://angularprimitives.com/getting-started/introduction

**Comment by ashley-hunter** [comment]
Thank you for considering Angular Primitives! If there is anything I can do to help just let me know! I think something like this would be a great way to collect feedback on our primitives and help improve them!

**Reply by geromegrignon** [reply]
I have currently 3 milestones before using Angular Primitives:
- complete Community onboarding setup (issue templates, contributing guides..)
- creation of the API which will be used among the example apps
- the basic Angular template to be used for example apps

Once finished, I'll create an example based on the template.
It'll surely take some time but yeah, we are getting started so it'll happen!
